paper-toggle-button.css

```diff
@@ -22,26 +22,32 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 
 :host .toggle-bar {
   background-color: var(--paper-toggle-button-unchecked-bar-color, #000000);
+  @apply(--paper-toggle-button-unchecked-bar);
 }
 
 :host .toggle-button {
   background-color: var(--paper-toggle-button-unchecked-button-color, --paper-grey-50);
+  @apply(--paper-toggle-button-unchecked-button);
 }
 
 :host([checked]) .toggle-bar {
   background-color: var(--paper-toggle-button-checked-bar-color, --google-green-500);
+  @apply(--paper-toggle-button-checked-bar);
 }
 
 :host([checked]) .toggle-button {
   background-color: var(--paper-toggle-button-checked-button-color, --google-green-500);
+  @apply(--paper-toggle-button-checked-button);
 }
 
 :host .toggle-ink {
   color: var(--paper-toggle-button-unchecked-ink-color, --primary-text-color);
+  @apply(--paper-toggle-button-unchecked-ink);
```

Comment:
Is there anything other than the colour you want to set for the ink? I think `--paper-toggle-button-unchecked-ink` and `--paper-toggle-button-checked-ink` won't get a lot of use